server/channels/db/migrations/postgres/000110_create_desktop_tokens.up.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE IF NOT EXISTS desktoptokens (
+    desktoptoken VARCHAR(64) NOT NULL,
+    servertoken VARCHAR(64),
+    userid VARCHAR(26),
+    createat BIGINT NOT NULL,
+    PRIMARY KEY (desktoptoken)
+);
+
+CREATE INDEX IF NOT EXISTS idx_desktoptokens_desktoptoken_servertoken_createat ON desktoptokens(desktoptoken, servertoken, createat)
```

Comment:
`CREATE INDEX CONCURRENTLY` for postgres: https://developers.mattermost.com/contribute/more-info/server/schema-migration-guide/#details

Reply:
```suggestion
CREATE INDEX CONCURRENTLY IF NOT EXISTS idx_desktoptokens_desktoptoken_servertoken_createat ON desktoptokens(desktoptoken, servertoken, createat)
```

Reply:
@agnivade Adding this causes tests to fail. Going to remove for now, is my syntax wrong? It looks like it's supposed to work: https://www.postgresql.org/docs/current/sql-createindex.html